modules/log6x-code-samples.adoc

```diff
@@ -0,0 +1,119 @@
+// Module included in the following assemblies:
+//
+// observability/logging/log6x-clf.adoc
+
+
+:_mod-docs-content-type: REFERENCE
+[id="log6x-code-samples"]
+= Code Samples
+
+[source,yaml]
+----
+# Metadata Section: Defines the basic identification of the resource
+apiVersion: observability.openshift.io/v1
+kind: ClusterLogForwarder
+metadata:
+  name: logging  # Name of the ClusterLogForwarder resource
+  namespace: openshift-logging  # Namespace where the resource is located
+
+# Spec Section: Configures the audit policy with rules for logging
+rules:
+  # Rule Section: Specifies the actions that should be logged
+  - level: Metadata  # Log request metadata (including URI, verb, and user) for matched requests
+    verbs: ["create", "update", "patch", "delete"]  # Specify the HTTP verbs to audit
+    resources:
+      - group: ""  # Empty string refers to the core API group
+        resources: ["pods"]  # Apply the rule to pod resources
+    namespaces: ["default", "kube-system"]  # Limit the rule to specific namespaces
+    omitStages: ["RequestReceived"]  # Exclude specific audit stages from logging
+
+  # Another Rule Section: Logging all other requests at the 'None' level
+  - level: None  # Do not log requests that do not match any other rules
+
+# OmitStages Section: Specifies stages of request processing to be excluded from audit logs
+omitStages:
+  - "RequestReceived"  # Exclude the initial request receipt from audit logs
+----
+
+
+[source,yaml]
+----
+apiVersion: observability.openshift.io/v1
+kind: ClusterLogForwarder
+metadata:
+  name: logging
+  namespace: openshift-logging
+
+spec:
+  collector:
+    nodeSelector:
+      kubernetes.io/os: linux 
+
+# Filters Section: Define filters to process logs
+  filters:
+  - name: add-label             # <1>
+    openShiftLabels:
+      custom-labels-added: "true"  # <2>
+    type: openShiftLabels          # <3>
+
+# Inputs Section: Specify sources of logs
+  inputs:
+  - application:
+      includes:
+      - namespace: project-qa-1    # <4>
+      - namespace: project-qa-2    # <5>
+      selector:
+        matchLabels:
+          test: centos-logtest     # <6>
```

Comment:
Probably, we should change centos for something being more like "rhel-logtest"?

Reply:
Sorry, I should have commented this file so that it was clear it was just to show how the combination of callouts and inline YAML comments to separate sections  would look.